guides/common/header.adoc

```diff
@@ -10,6 +10,13 @@ endif::[]
 ifeval::["{DocState}" == "nightly"]
 :revnumber: Nightly
 :revdate: published {date_my}
+++++
+<!-- "Fork me on GitHub" CSS ribbon v0.2.3 | MIT License -->
+<!-- https://github.com/simonwhitaker/github-fork-ribbon-css -->
+<a class="github-fork-ribbon right-top fixed" data-ribbon="Pre-release version" title="Pre-release version">
+Pre-release version
+</a>
+++++
```

Comment:
Have you tried adding the code to [docinfo.html](https://github.com/theforeman/foreman-documentation/blob/master/guides/common/docinfo.html) instead? Do you mind trying? :)

Reply:
I'd prefer to keep it here because this way I can leverage the existing "nightly" and "unsupported" conditions.